pkgs/development/tools/gauge/plugins/default.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, pkgs }:
+lib.makeScope pkgs.newScope (final: {
+  makeGaugePlugin = final.callPackage ./make-gauge-plugin.nix { };
+  dotnet = final.callPackage ./dotnet { };
```

Comment:
Please use with or inherit to avoid the repeated finals.

Reply:
Imho there is no problem being explicit here.

however `with` is bad.